src/installation/guides/chroot.md

```diff
@@ -0,0 +1,303 @@
+UUID guarantees they will be found even if they are assigned a different name at
+a later time. In some situations, such as booting from USB, this is absolutely
+essential. In other situations, disks will always have the same name unless
+drives are physically added or removed. Therefore, this step may not be strictly
+necessary, but is almost always recommended.
+
+Change the last zero of the entry for `/` to `1`, and the last zero of every
+other line to `2`. These values configure the behaviour of
+[fsck(8)](https://man.voidlinux.org/fsck.8).
+
+For example, the partition scheme used throughout previous examples yields the
+following `fstab`:
+
+```
+/dev/sda1       /boot/EFI   vfat    rw,relatime,[...]       0 0
+/dev/sda2       /           ext4    rw,relatime             0 0
+```
+
+The information from `blkid` results in the following `/etc/fstab`:
+
+```
+UUID=6914[...]  /boot/EFI   vfat    rw,relatime,[...]       0 2
+UUID=dc1b[...]  /           ext4    rw,relatime             0 1
+```
+
+Note: The output of `/proc/mounts` will have a single space between each field.
+The columns are aligned here for readability.
+
+Add an entry to mount `/tmp` in RAM:
+
+```
+tmpfs           /tmp        tmpfs   defaults,nosuid,nodev   0 0
+```
+
+If using swap space, add an entry for any swap partitions:
+
+```
+UUID=1cb4[...]  swap        swap    rw,noatime,discard      0 0
+```
+
+## Installing GRUB
+
+Use
+[grub-install](https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html)
+to install GRUB onto your boot disk.
+
+**On a BIOS computer**, install the package `grub`, then run `grub-install
+/dev/sdX`, where `/dev/sdX` is the drive (not partition) that you wish to
+install GRUB to. For example:
+
+```
+(chroot) # xbps-install grub
+(chroot) # grub-install /dev/sda
+```
+
+**On a UEFI computer**, install either `grub-x86_64-efi` or `grub-i386-efi`,
+depending on your architecture, then run `grub-install`, optionally specifying a
+bootloader label (this label may be used by your computer's firmware when
+manually selecting a boot device):
+
+```
+(chroot) # xbps-install grub-x86_64-efi
+(chroot) # grub-install --bootloader-id="Void"
+```
+
+If installing onto a removable disk (such as USB), add the option `--removable`
+to the `grub-install` command.
+
+## Finalization
+
+Use [xbps-reconfigure(1)](https://man.voidlinux.org/xbps-reconfigure.1) to
+ensure all installed packages are configured properly:
+
+```
+(chroot) # xbps-reconfigure -fa
+```
+
+This will make [dracut(8)](https://man.voidlinux.org/dracut.8) generate an
+initramfs, and will make GRUB generate a working configuration.
+
+At this point, the installation is complete. Exit the chroot and reboot your
+computer:
+
+```
+(chroot) # exit
+# shutdown -r now
+```
+
+After booting into your Void installation for the first time, be sure to do a
+system [update](../../xbps/updating.md).
```

Comment:
Final nitpick, I guess: "perform a system update". I'm excited to merge this, thanks a lot for the effort!

Reply:
I copied that line out of installation/live-images/guide.md. I prefer "perform a system update as well". Can I change both?

Reply:
Sure!

Reply:
this is ready to go